src/main/java/com/google/api/codegen/go/GoGapicContext.java

```diff
@@ -544,7 +547,7 @@ public boolean hasEmptyRetryCodes(Interface service, MethodConfig methodConfig)
    * Returns true if the API has a page streaming method.
    */
   public boolean hasPageStreamingMethod() {
-    for (Interface service : getModel().getSymbolTable().getInterfaces()) {
+    for (Interface service : new InterfaceView().getElementIterable(getModel())) {
```

Comment:
I couldn't think of anything that would be a significant improvement over what I have here.